tools/cli/commands/generate.js

```diff
@@ -378,7 +378,19 @@ async function generatePluginFromStarter( projDir, answers ) {
 	files = files.split( '\n' ).map( str => str.replace( 'projects/plugins/starter-plugin', '' ) );
 	files.forEach( file => {
 		if ( file && ! file.startsWith( 'changelog/' ) ) {
-			copyFile( path.join( projDir, file ), path.join( starterDir, file ) );
+			if ( file === 'tests/php/Jetpack_Starter_Plugin_Test.php' ) {
+				copyFile(
+					path.join(
+						projDir,
+						'tests/php/' + transformToPhpClassName( answers.name, false ) + '_Test.php'
+					),
+					path.join( starterDir, file )
+				);
+			} else if ( file === 'CHANGELOG.md' ) {
+				copyFile( path.join( projDir, file ), 'tools/cli/skeletons/common/CHANGELOG.md' );
+			} else {
+				copyFile( path.join( projDir, file ), path.join( starterDir, file ) );
```

Comment:
Looks like we need something like this here
```suggestion
			} else if ( fs.lstatSync( path.join( starterDir, file ) ).isSymbolicLink() ) {
				copySymlink( path.join( projDir, file ), path.join( starterDir, file ) );
			} else {
				copyFile( path.join( projDir, file ), path.join( starterDir, file ) );
```
(seems I missed this code path in #42266)

Reply:
I'll have you know I did check if the files were created...but I failed to check if they were symlinks. 🙃 c3d0d1089e